outputs/Console.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace GameOfLife\outputs;
+
+use GameOfLife\Field;
+use Ulrichsg\Getopt;
+
+/**
+ * Creates Console-output
+ * Shows the output on console.
+ * @package GameOfLife\outputs
+ */
+class Console extends BaseOutput
+{
+    /**
+     * Allows to add options to the variable $options in the gameoflife.php file.
+     * @param Getopt $_options
+     */
+    public function addOptions(Getopt &$_options)
+    {
+
+    }
+
+    /**
+     * Starts the output.
+     * @param Getopt $_options
+     */
+    public function startOutput(Getopt $_options)
+    {
+
+    }
+
+    /**
+     * Outputs the field.
+     * @param Field $_field
+     */
+    public function outputField(Field $_field)
+    {
+        $_field->printField();
```

Comment:
Now that you refactored output into its own classes, the printField() method should not be part of the field-class anymore. Please move the implementation of printField into this class and remove the printField() afterwards from the field-class. (Remember, the goal is that the field-class gets smaller in every step. :-))

Reply:
Right now, the function printField() is also used in the userInput class and relies on the Field class to function. If I would move printField() to the Console class, my input would depend on my output. So, I am not sure if it does make a lot sense right now...

Reply:
As discussed, please move the code and use the Console-Output in the UserInput class to print the field.